src/components/Navigation/Footer.js

```diff
@@ -0,0 +1,135 @@
+import React from 'react';
+import { NavLink } from 'react-router-dom';
+import { makeStyles } from '@material-ui/core/styles';
+import './Footer.css';
+import greenLogo from '../../assets/images/F2P-green-logo.svg';
+
+const useStyles = makeStyles({
+  root: {
+    display: 'flex',
+    lineHeight: 1.5,
+    flexWrap: 'wrap',
+    marginTop: '1rem',
+    padding: '1rem',
+    backgroundColor: '#EBD7D0',
+    position: 'relative',
+    width: '100%',
+  },
+  footerLogo: {
+    paddingLeft: '8%',
+    paddingRight: '16%',
+    alignSelf: 'center',
+  },
+  footerItem: {
+    fontFamily: 'Work Sans',
+    textDecoration: 'none',
+    fontSize: '14px',
+    color: '#373737',
+  },
+  footerItemTitle: {
+    fontFamily: 'Work Sans',
+    textDecoration: 'none',
+    fontSize: '14px',
+    color: '#373737',
+    fontWeight: 'bold',
+  },
+
+  footerList: {
+    listStyleType: 'none',
+
+  },
+  footerLink: {
+    color: '#000000',
+    textDecoration: 'none',
+    height: '100%',
+    '&:hover': {
```

Comment:
Is it possible to have some sort of visual feedback similar to this when clicking on the logo to navigate home? It might be helpful

Reply:
I just talked to Viv, and she prefers no feedback for the logos